src/app/gene-plot/gene-plot.ts

```diff
@@ -10,7 +10,9 @@ export class GenePlotModel {
 
   constructor(gene: Gene, rangeWidth: number, spacerLength: number = 150) {
     this.gene = gene;
-    this.spacerLength = spacerLength; // in px
+    this.spacerLength = spacerLength - (gene.chromosomes.size >= 3 ? (gene.transcripts.length >= 25 ? (gene.transcripts.length <= 50 ? gene.transcripts.length : 50) * 2.5 : 0) : 0);
```

Comment:
Use only `gene.chromosomes.size` to determine spacerLength, remove the check for the amount of transcripts